example/src/Examples/TextInputExample.tsx

```diff
@@ -444,6 +452,46 @@ const TextInputExample = () => {
             label="Custom rounded input"
           />
         </View>
+        <View style={styles.inputContainerStyle}>
```

Comment:
Is this View needed (the same style is passed to the input's style)?

Reply:
Removed